linkerd/app/core/src/metrics.rs

```diff
@@ -96,6 +96,14 @@ pub struct RouteAuthzLabels {
     pub authz: Arc<policy::Meta>,
 }
 
+/// Labels referencing an inbound HTTP local rate limit.
+#[derive(Clone, Debug, Eq, PartialEq, Hash)]
+pub struct HTTPLocalRateLimitLabels {
```

Comment:
Let's keep this in the app-inbound crate -- we should generally avoid adding things to the core crate if we can avoid it.